src/library/browse/browsetablemodel.h

```diff
@@ -37,35 +37,42 @@ const int COLUMN_REPLAYGAIN = 20;
 // The BrowseTable models displays tracks
 // of given directory on the HDD.
 // Usage: Recording and Browse feature.
+//
+// NOTE(uklotzde): Accessing tracks from the browse view will
```

Comment:
I will delete the question and move the comment (including your remarks) to 
the corresponding invocation to reveal this intention.

On 12/27/2015 05:49 PM, RJ Ryan wrote:
>
> In src/library/browse/browsetablemodel.h 
> <https://github.com/mixxxdj/mixxx/pull/728#discussion_r48456406>:
>
> > @@ -37,35 +37,42 @@ const int COLUMN_REPLAYGAIN = 20;
> >  // The BrowseTable models displays tracks
> >  // of given directory on the HDD.
> >  // Usage: Recording and Browse feature.
> > +//
> > +// NOTE(uklotzde): Accessing tracks from the browse view will
>
> This was intentional at the time since some people use Browse instead of 
> the library and we want to let them edit the tracks in a way that persists 
> across sessions and we didn't want to edit the files on disk by default 
> unless the user opts in to that.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/mixxxdj/mixxx/pull/728/files#r48456406>.
>